include/circt/Dialect/MSFT/MSFTOpInterfaces.td

```diff
@@ -0,0 +1,27 @@
+//===- MSFTOpInstances.td - MSFT OpInstance definitions ----*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+include "mlir/IR/OpBase.td"
+
+def InstanceSpecificOpInterface : OpInterface<"InstanceSpecificOpInterface"> {
```

Comment:
nitpicking on the name, I don't quite see what the name means about "instance-specific". I'm not sure what else you plan to put here, but I guess it will be for things that a dynamic instance targets? Maybe call it something with "dynamic instance" in the name? Or, the description seems to point directly to this being global ref related, so maybe call this something like "GlobalRefTargetOpInterface" or something?

Reply:
My gut tells me that there will eventually be stuff other than set global ref. I think I just viscerally don't like "dynamic instance", so I keep unconscientiously inventing new synonyms. Changed.